ui-admin/src/study/participants/participantList/ParticipantList.test.tsx

```diff
@@ -29,6 +31,35 @@ test('renders a participant with link', async () => {
   expect(participantLink).toHaveAttribute('href', `/${studyEnvContext.currentEnvPath}/participants/JOSALK`)
 })
 
+test('renders filters for participant columns', async () => {
+  const studyEnvContext = mockStudyEnvContext()
+  const { RoutedComponent } = setupRouterTest(<ParticipantList studyEnvContext={studyEnvContext}/>)
+  render(RoutedComponent)
+
+  //Assert that all 3 default columns have filter inputs
+  const searchInputs = await screen.findAllByPlaceholderText('Search...')
+  expect(searchInputs).toHaveLength(3)
+})
+
+test('filters participants based on shortcode', async () => {
+  const studyEnvContext = mockStudyEnvContext()
+  const { RoutedComponent } = setupRouterTest(<ParticipantList studyEnvContext={studyEnvContext}/>)
+  render(RoutedComponent)
+
+  //Assert that JOSALK is visible in the table
+  await screen.findByText('JOSALK')
+
+  //Search for some unknown shortcode
+  await act(() =>
+    userEvent.type(screen.getAllByPlaceholderText('Search...')[0], 'UNKNOWN SHORTCODE')
+  )
+
+  //Assert that JOSALK is no longer visible in the table
+  await waitFor(() => {
+    expect(screen.queryByText('JOSALK')).not.toBeInTheDocument()
+  })
+})
```

Comment:
nice test.  This could also use a positive test case -- that if you type "jos" JOSALK is still visible